src/Entities/CompleteApplicantEntity.php

```diff
@@ -36,6 +36,20 @@ class CompleteApplicantEntity extends ApplicantEntity implements \JsonSerializab
     protected $additionalNotes;
     protected $chosenCourseId;
     protected $chosenCourseDate;
+    protected $attitude;
```

Comment:
The others aren't typehinted so I don't know why these would be but should these be typehinted?

Reply:
wait is it because they return mixed?

Reply:
but why do they return mixed?

Reply:
shhhh